content/06.methods.md

```diff
@@ -229,8 +229,9 @@ All p-values are two-sided unless otherwise stated.
 Z-scores were calculated using the formula $z=(x –\mu)/\sigma$ where $x$ is the value of interest, $\mu$ is the mean, and $\sigma$ is the standard deviation.
 
 #### Tumor purity (`tumor-purity-exploration` module)
-We used Theta2 (as described in the "Somatic Copy Number Variant Calling section" Methods section) to infer tumor purity of WGS samples and explored purity values across histologies. 
-We assumed that co-extracted RNA and DNA samples had the same tumor purity.
+Estimating tumor fraction from RNA directly is challenging since methods focus on estimating proportion of immune cells and assuming the rest is tumor [@doi:10.1038/ncomms3612], which is not a fair assumption for many of these diagnoses.
+We therefore used Theta2 (as described in the "Somatic Copy Number Variant Calling section" Methods section) to infer tumor purity of WGS samples and assumed that co-extracted RNA and DNA samples had the same tumor purity.
+We then created a set of stranded RNA-Seq data thresholded by median tumor purity of the cancer group to rerun selected transcriptomic analyses: `telomerase-activity-prediction`, `tp53_nf1_score`, `transcriptomic-dimension-reduction`, `immune-deconv`, and `gene-set-enrichment-analysis`. 
```

Comment:
I added an additional sentence to be explicit about which samples were used, and then I added a bit about module location specifically because this is in a `tumor-purity-exploration` subsection. Feel free to rephrase.

```suggestion
We then created a set of stranded RNA-Seq data thresholded by median tumor purity of the cancer group to rerun selected transcriptomic analyses: `telomerase-activity-prediction`, `tp53_nf1_score`, `transcriptomic-dimension-reduction`, `immune-deconv`, and `gene-set-enrichment-analysis`. 
Note that these thresholded analyses, which only considered stranded RNA samples that also had co-extracted DNA, were performed in their respective OpenPBTA analyses modules (not within `tumor-purity-exploration`).
```